.github/workflows/docs.yaml

```diff
@@ -0,0 +1,14 @@
+name: Documentation Updates
+
+on: [push]
+
+jobs:
+    build-html:
+        name: Validate & Build HTML
+        runs-on: ubuntu-18.04
+        steps:
+            - uses: actions/checkout@v1
+            - uses: docker://errordeveloper/cilium-docs:latest
```

Comment:
I pushed my own image for now, let's discuss where this one should live and if we need to tag and push it automatically.

Reply:
For sure, I just need push access to begin with :)

I'd be happy to automate pushing the image, but perhaps best to do it in another PR?

Reply:
Usually I think they just pull from repo updates and build on the remote infrastructure, is there something specific we need to automate here?

Reply:
Ah, maybe we only do this for the main cilium image so if the docs base target doesn't change often then we don't need to keep pushing a new image.

Reply:
Yes, this one shouldn't change very often.